README.md

```diff
@@ -1,47 +1,166 @@
 # MN Bootstrap
 
-> Distribution package for Dash Masternode installation
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+Distribution package for Dash Masternode installation
 
 ## Table of Contents
 
-- [Pre-requisites](#Pre-requisites)
+- [Pre-requisites](#pre-requisites)
```

Comment:
Do we still have this section?

Reply:
No, looks like that section was removed. I deleted it in b2ea9c6.